src/components/pager/__spec__.js

```diff
@@ -114,6 +114,13 @@ describe('Pager', () => {
       });
     });
 
+    describe('when element is not incorrect', () => {
```

Comment:
`when element is not incorrect it does not call onPagination` contains three negatives, can it be reworded to be clearer?